src/tests/itests/iotc_itests.c

```diff
@@ -48,27 +48,9 @@ int iotc_itests_main(int argc, char const* argv[])
 {
   iotc_test_init(argc, argv);
 
-  int i;
-  char host_name[2048];
-	uint16_t port_val = 0;
-
-  iotc_test_init(argc, argv);
-
-  	for (i=1; i<argc; ++i) {
-		if (argv[i][0] == '-') {
-			if (!strcmp(argv[i], "--hostURL")) {
-				i++;
-				memcpy(host_name, argv[i], strlen(argv[i]));
-			} else if (!strcmp(argv[i], "--hostPort")) {
-				i++;
-				port_val = atoi(argv[i]);
-			}
-		}
-	}
-	if(port_val > 0){
-		/* caller supplied custom hostURL and port number  */
-		setHostNameAndPort(host_name, port_val);
-	}
+  /* caller supplied custom hostURL and port number  */
+  /* Example: */
+  /*setHostNameAndPort("us-central1-mqtt.clearblade.com", "443");*/
```

Comment:
I'd like to change this example to something like `mycustomdomain-mqtt.clearblade.com` for the host name. If a customer is using our default region based URL they wouldn't need to call this function at all typically, as the region specific host name is dynamically determined based on client id.